gui.c

```diff
@@ -100,7 +100,10 @@ GtkWidget *StripGPSButton;
 GtkWidget *PhotoListVBox;
 GtkWidget *PhotoListScroll;
 GtkWidget *PhotoList;
+#if GTK_CHECK_VERSION(2, 12, 0)
```

Comment:
This would be slightly cleaner as #if !GTK_CHECK_VERSION(2, 12, 0)

Reply:
Fixed.